srcpkgs/apparmor/template

```diff
@@ -1,12 +1,13 @@
 # Template file for 'apparmor'
 pkgname=apparmor
-version=3.0.3
-revision=4
+version=3.1.2
+revision=1
 build_wrksrc=libraries/libapparmor
 build_style=gnu-configure
 conf_files="/etc/apparmor.d/local/* /etc/apparmor/*"
 make_dirs="/etc/apparmor.d/disable 0755 root root"
-hostmakedepends="bison flex autoconf automake libtool gettext swig python3 which"
+hostmakedepends="bison flex autoconf automake libtool gettext swig python3 which
```

Comment:
iirc, this version doesn't call which anymore, would you mind checking if that's correct?
```suggestion
hostmakedepends="bison flex autoconf automake libtool gettext swig python3
```

Reply:
`which` still seems to be required, as the build fails with this error without it:
```
config.status: error: in `/builddir/apparmor-3.1.2/libraries/libapparmor':
config.status: error: Something went wrong bootstrapping makefile fragments
    for automatic dependency tracking.  If GNU make was not used, consider
    re-running the configure script with MAKE="gmake" (or whatever is
    necessary).  You can also try re-running configure with the
    '--disable-dependency-tracking' option to at least be able to build
    the package (albeit without support for automatic dependency tracking).
See `config.log' for more details
```
Opening `masterdir/builddir/apparmor-3.1.2/libraries/libapparmor/config.log` and searching for `which` then brings me to this:
```
config.status:1302: cd src       && sed -e '/# am--include-marker/d' Makefile         | make -f - am--depfiles
make: which: No such file or directory
../../../common//Make.rules:40: *** awk utility required for build but not available.  Stop.
config.status:1307: $? = 2
config.status:1302: cd testsuite       && sed -e '/# am--include-marker/d' Makefile         | make -f - am--depfiles
make: which: No such file or directory
../../../common/Make.rules:40: *** awk utility required for build but not available.  Stop.
config.status:1307: $? = 2
config.status:1311: error: in `/builddir/apparmor-3.1.2/libraries/libapparmor':
config.status:1313: error: Something went wrong bootstrapping makefile fragments
    for automatic dependency tracking.  If GNU make was not used, consider
    re-running the configure script with MAKE="gmake" (or whatever is
    necessary).  You can also try re-running configure with the
    '--disable-dependency-tracking' option to at least be able to build
    the package (albeit without support for automatic dependency tracking).
See `config.log' for more details
```

Reply:
and for runtime?

Reply:
Removing `which` from runtime deps seems to work. `which` is pulled in anyway by `base-system` though, so it's not like it matters whether it's specified as a runtime dep or not, is that right?

Reply:
That's true, all base-* packages depend on which, but I would still explicitly depend on it exactly where it's necessary.

Reply:
I searched and found https://gitlab.com/apparmor/apparmor/-/merge_requests/784, included in AppArmor 3.1.1, so I guess `which` is not a required runtime dependency anymore. I'll update the PR accordingly.